.github/workflows/test-and-release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
```

Comment:
sorry, it's my first pull request and review. node-version 12.x is enough. how can I delete my change?